org.eclipse.wildwebdeveloper/pom.xml

```diff
@@ -78,7 +78,7 @@
 						<configuration>
 							<executable>npm</executable>
 							<arguments>
-								<arg>ci</arg>
+								<arg>install</arg>
```

Comment:
Is there a reason to use `npm install` here instead of  `npm ci`? According to the documentation `npm ci`  is similar to `npm install` command. Differences are: 

-  The project must have an existing package-lock.json or npm-shrinkwrap.json .
-  If dependencies in the package lock do not match those in package.json , npm ci will exit with an error, instead of updating the package lock.
-  npm ci can only install entire projects at a time: individual dependencies cannot be added with this command.
-  If a node_modules is already present, it will be automatically removed before npm ci begins its install.
-  It will never write to package.json or any of the package-locks: installs are essentially frozen.

I know there is some other differences like this one https://npm.community/t/npm-ci-no-bin-links-is-not-honored/9180 but I'm think we should rely on `npm ci` command because dependencies are frozen.

Reply:
I changed exactly because it's not updating package-lock thus when changing versions package.json and package-lock.json get out of sync. It's probably good idea to change it back to ci when things are working.